zproject/urls.py

```diff
@@ -780,7 +780,9 @@
 
 # User documentation site
 help_documentation_view = MarkdownDirectoryView.as_view(
-    template_name="zerver/documentation_main.html", path_template="/zerver/help/%s.md"
+    template_name="zerver/documentation_main.html",
+    path_template="/%s.md",
+    help_view=True,
```

Comment:
I'm not a fan of adding this boolean and this `path_template`, but I wanted the directory for the template engine to be the help directory.

Reply:
Yeah, this is a bit weird -- maybe just add a brief comment noting that `path_template` is the path starting from the root of the system, which `help_view=True` configures to be `/help/`?

Reply:
Okay. I realized that we want to be handling these pages more like the policy documentation pages in `render_markdown_path`. So I went ahead and updated the `path_template` to be the absolute path here.

An alternative would be to only set `help_view=True` here and then set the `path template` in `MarkdownDirectoryView.get` like we do with the policy documentation.

Reply:
I think either way is probably fine for now. Once we migrate the API docs out of `templates/zerver`, it may be more clear what is cleanest.